spec/controllers/application_controller/automate_spec.rb

```diff
@@ -54,7 +54,7 @@ def resolve_hash(button)
         controller.instance_variable_set(:@sb, sb)
         controller.params = {:button => 'throw'}
         allow(MiqAeEngine).to receive(:resolve_automation_object).and_return(workspace)
-        data = "<MiqAeWorkspace>\\n<MiqAeObject namespace='ManageIQ/SYSTEM'>\\n</MiqAeObject>\\n</MiqAeWorkspace>\\n"
+        data = "<MiqAeWorkspace>\\n<MiqAeObject namespace='ManageIQ/SYSTEM'>\\n</MiqAeObject>\\n</MiqAeWorkspace>"
```

Comment:
can we change `"\\n"` to `"\n"` (and leave in the trailing one)

for both of these?